hr_employee_data_from_work_address/README.rst

```diff
@@ -0,0 +1,37 @@
+User data from employee
+=======================
+
+When using HR, there are a couple of partners involved per employee: The employee's work address, the employee's private address and the partner assigned to the employee's user (if any). The latter is used to store some data about the user, like the email address. In many cases, it would be more convenient to edit this data from the employee record. This module assumes you assign a (different) partner for the work address to every employee, and synchronizes its values with the employee record. Then you should use this partner record also for the user record.
+
+Usage
+=====
+
+After installation, updating an employee's `work_email` or `work_phone` record transparently changes the linked partner's appropriate field. For obvious reasons, the default for the work address being the company's address has been lifted. Partners created through the work address field will have the `employee`-flag set, and the `partner_id` field on the user record filters for this flag.
```

Comment:
I agree with the functionality provided, and I noticed this assumption. However, as setting the company address is the default in Odoo, you might want to make this into a big warning saying that things get ugly quickly if your employees actually do have the company address (this will lead to random employees' email addresses and phone numbers appear on your sale orders, and users being created from such employees forms get the company address)